src/cpx_constrs.jl

```diff
@@ -494,12 +494,12 @@ function add_indicator_constraint(model::Model, idx::Vector{Cint}, coeff::Vector
                       Cint,
                       Cint,
                       Cdouble,
-                      Cchar,
+                      Cint,
```

Comment:
I understand that the [docs say this is a `Cint`](https://www.ibm.com/support/knowledgecenter/SSSA5P_12.10.0/ilog.odms.cplex.help/refcallablelibrary/gcapi/addindconstr.html), but this is surely a mistake. What would this be the only place in the API where the sense is a `Cint`. What happens if you make this a `Cchar`? Does it work? 

I can follow up with the developers.

Reply:
I'll re-test with Cchar to see if it works

Reply:
I've mailed the CPLEX developers for clarification.

Reply:
seems all fine, I reverted to Cchar

Reply:
Here's the response from CPLEX:

<img width="565" alt="image" src="https://user-images.githubusercontent.com/8177701/75373108-44704f00-588f-11ea-812c-190224913e53.png">

Please change to `Cint` and leave a comment.

Reply:
done thanks for reaching out to them